src/app/access-requests/features/access-request-manager-detail/access-request-manager-detail.component.html

```diff
@@ -233,65 +248,79 @@ <h2 mat-dialog-title>Access Request Detail</h2>
         </mat-card>
       </div>
     </div>
-  </mat-dialog-content>
-  <mat-dialog-actions class="flex gap-x-2 gap-y-4 border-gray-500 pt-4">
-    @if (ivasAreLoading() || ivasError() || !ivas().length || !changeable()) {
-      <button
-        mat-stroked-button
-        (click)="cancel()"
-        data-umami-event="Access Request Manager Close Clicked"
-      >
-        Close
-      </button>
-    } @else {
-      <div class="w-full sm:w-auto">
+
+    <div class="mt-8 flex">
+      @if (ivasAreLoading() || ivasError() || !ivas().length || !changeable()) {
         <button
-          autoFocus
           mat-stroked-button
-          class="w-full sm:w-auto"
-          (click)="cancel()"
-          data-umami-event="Access Request Manager Cancel Clicked"
+          (click)="goBack()"
+          data-umami-event="Access Request Manager Close Clicked"
         >
-          Cancel
+          Close
         </button>
-      </div>
-      <div class="flex-grow"></div>
-      <div class="flex w-full flex-wrap gap-2 sm:w-auto sm:flex-nowrap">
+      } @else {
         <div class="w-full sm:w-auto">
           <button
-            mat-raised-button
-            class="error w-full sm:w-auto"
-            (click)="saveBeforeStatusChange('deny')"
-            data-umami-event="Access Request Manager Deny Clicked"
-            [disabled]="data.status === 'denied'"
+            autoFocus
+            mat-stroked-button
+            class="w-full sm:w-auto"
+            (click)="goBack()"
+            data-umami-event="Access Request Manager Close Clicked"
           >
-            @if (data.status === 'allowed') {
-              Revoke access
-            } @else {
-              Deny
-            }
+            Close
```

Comment:
```suggestion
            Cancel
```

Reply:
Already handled in the conversion above.